.prettierrc.json

```diff
@@ -1,16 +1,16 @@
 {
-	"arrowParens": "avoid",
-	"bracketSpacing": true,
-	"htmlWhitespaceSensitivity": "css",
-	"insertPragma": false,
-	"jsxSingleQuote": false,
-	"printWidth": 100,
-	"proseWrap": "always",
-	"quoteProps": "as-needed",
-	"requirePragma": false,
-	"semi": true,
-	"singleQuote": false,
-	"tabWidth": 4,
-	"trailingComma": "all",
-	"useTabs": true
+    "arrowParens": "avoid",
+    "bracketSpacing": true,
+    "htmlWhitespaceSensitivity": "css",
+    "insertPragma": false,
+    "jsxSingleQuote": false,
+    "printWidth": 100,
+    "proseWrap": "always",
+    "quoteProps": "as-needed",
+    "requirePragma": false,
+    "semi": true,
+    "singleQuote": false,
+    "tabWidth": 4,
+    "trailingComma": "all",
+    "useTabs": false
```

Comment:
I hear people start fights over this, but this specific rule is actually useful in our repo. There is a whitespace eslint rule that throws errors if you mix spaces and tabs in the line-indentation. The formatter does this when defining multi-line union types for example, so I had to set this to spaces to avoid the whittspace mixing rule to go bananas.

Reply:
Hahah shit